tests/epyccel/modules/openmp.py

```diff
@@ -124,7 +124,7 @@ def test_omp_in_final():
 
     #$ omp parallel
     #$ omp single
-    #$ omp task final (i >= 10)
+    #$ omp task final(i >= 10)
```

Comment:
Is this change important? If I did `#$ omp task final (i >= 10)` would pyccel still understand?

Reply:
i didnt notice i changed it, there is no reason for this change I was just testing

Reply:
Ok :+1: I just wanted to check it wasn't something that needed documenting